examples/app/log_layers.rs

```diff
@@ -36,10 +36,24 @@ fn custom_layer(_app: &mut App) -> Option<BoxedLayer> {
     ]))
 }
 
+// While `custom_layer` allows you to add _additional_ layers, it won't allow you to override the
```

Comment:
I would add this doc in the LogPlugin as well, because I was initially confused as to the value that `fmt_layer` was bringin compared to `custom_layer`

Reply:
Resolved in [cbc54c1](https://github.com/bevyengine/bevy/pull/17722/commits/cbc54c133b700eab940a3607bd3bb428e5145a77).